examples/with-magic/README.md

```diff
@@ -40,11 +40,30 @@ Then set each variable on `.env.local`:
 
 - `NEXT_PUBLIC_MAGIC_PUBLISHABLE_KEY` should look like `pk_test_abc` or `pk_live_ABC`
 - `MAGIC_SECRET_KEY` should look like `sk_test_ABC` or `sk_live_ABC`
+- `TOKEN_SECRET` should be a string with at least 32 characters
 
-To deploy on Vercel, you need to set up the environment variables with the [Environment Variables UI](https://vercel.com/blog/environment-variables-ui) using the [Vercel CLI](https://vercel.com/download) ([Documentation](https://vercel.com/docs/cli#commands/env)).
-
-Install [Vercel CLI](https://vercel.com/download), log in to your account from the CLI, link your project and run the following command to add the `NEXT_PUBLIC_MAGIC_PUBLISHABLE_KEY` and `MAGIC_SECRET_KEY` environment variables.
+Now, run Next.js in development mode
 
 ```bash
-vercel env add
+npm run dev
+# or
+yarn dev
 ```
+
+Your app should be up and running on [http://localhost:3000](http://localhost:3000)! If it doesn't work, post on [GitHub discussions](https://github.com/vercel/next.js/discussions).
+
+## Deploy on Vercel
```

Comment:
Is this needed - isn't it the same as above?

Reply:
Same answer as for the comment above. We include a deploy button at the beginning of the example and do it again, but with more detailed instructions, at the end of the readme.